nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -566,6 +566,10 @@
 
 - The `services.patroni.raft` option has been removed, as Raft has been [deprecated by upstream since 3.0.0](https://github.com/patroni/patroni/blob/master/docs/releases.rst#version-300)
 
+- The `jd-gui` package was removed due to an inactive upstream and a dependency on the end-of-life Gradle 6.
+  Java decompilers already packaged in Nixpkgs include `jd-cli`, `cfr`,
```

Comment:
Only `bytecode-viewer` looks like it's a complete alternative, providing a GUI, others seem to only provide a cli interface, maybe this can be specified?